From debac6f9a266551e12443b90a7b3de9299596fe7 Mon Sep 17 00:00:00 2001
From: Daniel Drubin <daniel.drubin@intel.com>
Date: Sun, 16 Nov 2014 11:02:56 +0200
Subject: [PATCH] Fix for too early HECI connect requests - fixing style
 problems

Change-Id: Ib6b62df0e57cdb7a9887eeaddd8966d8ff18a4fd
Signed-off-by: Daniel Drubin <ddrubin@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-8839
Signed-off-by: Fabien Marotte <fabien.marotte@intel.com>
(cherry picked from commit c7cd38438ebc477feda0c1e81b3395233d42d084)
---
 drivers/misc/heci/hbm.c             |    2 --
 drivers/misc/heci/heci-hid-client.c |   10 +++++-----
 2 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/drivers/misc/heci/hbm.c b/drivers/misc/heci/hbm.c
index 3d4586d..1383b65 100644
--- a/drivers/misc/heci/hbm.c
+++ b/drivers/misc/heci/hbm.c
@@ -618,9 +618,7 @@ void heci_hbm_dispatch(struct heci_device *dev, struct heci_bus_message *hdr)
 			dev->print_log(dev, "%s(): HOST_CLIENT_PROPERTIES_RES_CMD, client# = %d props: ", __func__, dev->me_client_presentation_num);
 			for (i = 0; i < sizeof(struct hbm_props_response); ++i)
 				dev->print_log(dev, "%02X ", *(((unsigned char *)props_res) + i));
-			
 			dev->print_log(dev, "\n");
-			
 		} while (0);
 #endif
 
diff --git a/drivers/misc/heci/heci-hid-client.c b/drivers/misc/heci/heci-hid-client.c
index 78f1389..cec1d08 100644
--- a/drivers/misc/heci/heci-hid-client.c
+++ b/drivers/misc/heci/heci-hid-client.c
@@ -571,12 +571,12 @@ static int __init ish_init(void)
 	/* 7/7/2014: in order to not stick Android boot, from here & below needs to run in work queue and here we should return rv */
 	/****************************************************************/
 	workqueue_for_init = create_workqueue("workqueue_for_init");
-        INIT_WORK(&my_work, workqueue_init_function);
-        queue_work(workqueue_for_init, &my_work);
+	INIT_WORK(&my_work, workqueue_init_function);
+	queue_work(workqueue_for_init, &my_work);
+
+	ISH_DBG_PRINT(KERN_ALERT "[ish client driver] %s() enqueue init_work function\n", __func__);
+	g_ish_print_log(KERN_ALERT "[ish client driver] %s() enqueue init_work function\n", __func__);
 
-        ISH_DBG_PRINT(KERN_ALERT "[ish client driver] %s() enqueue init_work function\n", __func__);
-        g_ish_print_log(KERN_ALERT "[ish client driver] %s() enqueue init_work function\n", __func__);
-	
 	return rv;
 	/****************************************************************/
 
-- 
1.7.9.5

